tests/integration_tests/scheduler/bin/qstat.py

```diff
@@ -67,8 +67,9 @@ def main() -> None:
             user = "mock"
             time = "00:00:00"
             queue = "mocked"
+
             print(
-                f"{job:16.16}  {name:16.16} {user:16.16}  {time:8.8} {state:1.1} {queue:5.5}"
+                f"{job:30.30}  {name:15.15} {user:15.15}  {time:8.8} {state:1.1} {queue:5.5}"
```

Comment:
so "wide" only applies to job-name, but name and user will be shortened.. 😞

Reply:
But maybe that is okay, do we even use those fields for anything?

Reply:
It is okay yes :) To be solved later if a problem appears.